### ISSUE TITLE
Od 06 czerwca zwracamy identyfikator punktu odbioru - m.in. Poczty Polskiej

### ISSUE THREAD

**Issue by TomekKaczmarek** [allegro/allegro-api#264]
Od **06 czerwca** w metodach [doGetPostBuyFormsDataForBuyers ](https://allegro.pl/webapi/documentation.php/show/id,702)i [doGetPostBuyFormsDataForSellers ](https://allegro.pl/webapi/documentation.php/show/id,703)będziemy zwracać identyfikator punktu odbioru **postBuyFormAdrId (string)** w tablicach typu PostBuyFormAddressStruct. Pole postBuyFormAdrId pojawi się w tablicy postBuyFormGdAddress, jeśli kupujący wybrał punkt odbioru. 

Struktura wyjściowa tablicy postBuyFormGdAddress w tych metodach będzie wyglądała następująco:

```
postBuyFormGdAddress | PostBuyFormAddressStruct

1. postBuyFormAdrCountry | int 
   Identyfikator kraju.
2. postBuyFormAdrStreet | string 
   Adres punktu odbioru.
3. postBuyFormAdrPostcode | string 
   Kod pocztowy punktu odbioru.
4. postBuyFormAdrCity | string 
   Miasto, w którym znajduje się punkt odbioru.
5. postBuyFormAdrFullName | string 
   Nazwa punktu odbioru.
6. postBuyFormAdrCompany | string 
   Pole nieużywane (zawsze będzie puste).
7. postBuyFormAdrPhone | string 
   Telefon komórkowy użytkownika.
8. postBuyFormAdrNip | string
   Pole nieużywane (zawsze będzie puste).
9. postBuyFormCreatedDate | string 
   Data utworzenia formularza pozakupowego.
10. postBuyFormAdrType | int 
   Pole nieużywane (zawsze będzie puste).
11. postBuyFormAdrId | string
   Identyfikator punktu odbioru.
```
06 czerwca zaktualizujemy dokumentację metod doGetPostBuyFormsDataForBuyers i doGetPostBuyFormsDataForSellers.

**Comment by sb84** [comment]
Czy ta wartość została dodana do starego WSDLa?
Zwracam info o punkcie odbioru:

 [post-buy-form-gd-address] => stdClass Object
        (
            [post-buy-form-adr-country] => 1
            [post-buy-form-adr-street] => 3 Maja 28
            [post-buy-form-adr-postcode] => 22-440
            [post-buy-form-adr-city] => Krasnobród
            [post-buy-form-adr-full-name] => UP Krasnobród
            [post-buy-form-adr-company] => 
            [post-buy-form-adr-phone] => 
            [post-buy-form-adr-nip] => 
            [post-buy-form-created-date] => 2018-08-09 12:09:32
            [post-buy-form-adr-type] => 0
        )

i nie mam takiego pola

**Comment by TomekKaczmarek** [comment]
@sb84 Nie, dodaliśmy je tylko do nowego WSDLa: https://webapi.allegro.pl/service.php?wsdl

**Comment by sb84** [comment]
@TomekKaczmarek a dodacie to pole do starego WSDL'a ?

**Comment by TomekKaczmarek** [comment]
@sb84 nie rozwijamy go, ogólnie nie rozwijamy już WebAPI - planujemy je powoli wygaszać - https://github.com/allegro/allegro-api/issues/277
